virtio-queue/src/queue/verification.rs

```diff
@@ -458,19 +458,22 @@ impl kani::Arbitrary for ProofContext {
     }
 }
 
+/// # Specification (VirtIO 1.3, Section 2.7.7.2: "Device-to-driver notification suppression")
```

Comment:
In virtio 1.3 (https://docs.oasis-open.org/virtio/virtio/v1.3/csd01/virtio-v1.3-csd01.html#x1-510007) the section 2.7.7.2 is "2.7.7.2 Device Requirements: Used Buffer Notification Suppression", so I guess here we should update the title. (Is the 2.7.7.2 correct?)